src/app/app.module.ts

```diff
@@ -153,7 +157,7 @@ const NGA_TEST_COMPONENTS = [
     ...NGA_TEST_COMPONENTS,
   ],
   entryComponents: [
-    NgaDynamicToAddComponent,
+    NgaDynamicToAddComponent
```

Comment:
Should be a comma at the end, run `npm run lint:ci` to check out all of the code style issues.